content/en/docs/setup/install/operator/test.sh

```diff
@@ -39,4 +40,7 @@ _verify_contains snip_kubectl_get_svc "egressgateway"
 
 # @cleanup
 snip_cleanup
-kubectl delete ns istio-operator --grace-period=0 --force
+
+# Everything should be removed once cleanup completes. Use a small
+# timeout for comparing cluster snapshots before/after the test.
+export VERIFY_TIMEOUT=20
```

Comment:
nit: newline at the end

Reply:
this was addressed in 5d0ea40fd365c09ed4c42c04f5db56cf064efd9c